poc/pkg/templates/istio-gateway.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+   name: {{ .Values.istio.gateway.name }}
+   namespace: {{ .Values.istio.namespace }}
+spec:
+  selector:
+    istio: {{ .Values.istio.name }}
+  servers:
+  - port:
+      number: 443
+      name: http
+      protocol: HTTPS
+    tls:
+      mode: SIMPLE
```

Comment:
For now, could we disable HTTPS so we can test without this secret?

Reply:
I have changed the port from 443 to 80, we should able communicate without TLS now.